cloudformation/add_cloudflare_ips_to_sgs.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+# TODO: move this into Ansible or at least use Boto
```

Comment:
I'm okay checking this in but sorely tempted to rewrite it to use boto

Reply:
Done